benchmarks/migrations/0007_auto_20230309_1047.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.14 on 2023-03-09 15:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('benchmarks', '0006_model_competition'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='jenkins_id',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='submission',
+            name='id',
+            field=models.AutoField(primary_key=True, serialize=False),
+        ),
```

Comment:
shouldn't it alter `id` -> `jenkins_id` and add a new `id` for primary key? Possible it doesn't work this way; if it doesn't we'll need to move all `id` values into `jenkins_id`

Reply:
After trying, it is indeed odd to do it the first way, so I will do it the second way you mentioned. It is easier and faster